### DISCUSSION TITLE
How to create a seurat object list

### DISCUSSION THREAD

**Discussion by tang27abu** [satijalab/seurat#5014] [Q&A]
Hello,
I am trying to integrate multiple scRNA-Seq dataset with this guidance. https://satijalab.org/seurat/articles/integration_introduction.html

I have created multiple seurat objects using CreateSeuratObject respectively and I was wondering how can I make them a object list as same in the guidance. 
```
# split the dataset into a list of two seurat objects (stim and CTRL)
ifnb.list <- SplitObject(ifnb, split.by = "stim")

# normalize and identify variable features for each dataset independently
ifnb.list <- lapply(X = ifnb.list, FUN = function(x) {
    x <- NormalizeData(x)
    x <- FindVariableFeatures(x, selection.method = "vst", nfeatures = 2000)
})

# select features that are repeatedly variable across datasets for integration
features <- SelectIntegrationFeatures(object.list = ifnb.list)
```
I have obj1, obj2... but I dont know how to make them a in a list like shown in the code **ifnb.list.**

Thanks,
Zhen

**Comment by samuel-marsh** [comment]
Hi Zhen,

You can make a list of Seurat objects just like making list of anything in R.

```
obj_list <- list(Object_name1 = Object1, 
                 Object_name2 = Object2)
```

Best,
Sam
